### ISSUE TITLE
Real Estate Abatement Form

### ISSUE THREAD

**Issue by joshuagee** [CityOfBoston/Forms#414]
Contact: matthew.englander@boston.gov
phillip.cheetham@boston.gov

This form is for residents who want to trigger a reassessment of their property value. They think they city has charged them too much or too little and want an official new valuation. In reality, no one is saying, "I'm not paying enough" so it's mostly for people who think their property taxes are out of line with the current value of their house.

The form is in Liquid Office, and is triggered by a [search on the Assessing website](https://www.cityofboston.gov/assessing/search/
). That search starts an autofill, which determines if the person will be presented with the short version or the long version of the form. It also fills our the address and parcel information.  sure where that information is pulled from. There might be some kind of bar code that is also generated. 

On paper, these are one form. These online forms are hosted on Liquid Office. They **can't** be submitted online. The user fills them out and prints them. 

- The short version is for residential properties. You can find it online here: https://forms.cityofboston.gov/lfserver/AbatementShort
- The long version is for commercial properties. You can find it online here: https://forms.cityofboston.gov/lfserver/AbatementLong

After submitting the request, the customer has 30 days to also submit #12. That includes the information justifying the abatement. Many customers submit both at the same time.

**Comment by joshuagee** [comment]
Previously, I had this as one form (#14). Then, I split it into two because Commercial #15 and Residential #16 looked so different. I think it makes sense to merge it back into one.
